docs/features/chat-agent.md

```
@@ -1,532 +1,26 @@
 
-Chat agent for managing links via natural language, built on Cloudflare Agents SDK.
+AI chat for managing links via natural language, built on Cloudflare Agents SDK.
 
 ## Overview
 
-One chat per workspace with real-time WebSocket connection, message persistence, and LiveStore integration for link management.
+One chat per workspace with real-time WebSocket, message persistence in DO SQLite, and LiveStore integration for link management. Uses OpenRouter (`google/gemini-2.5-flash`) via Vercel AI SDK.
 
 ## Architecture
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)